treescript/src/treescript/versionmanip.py

```diff
@@ -126,7 +155,7 @@ def get_version(file_, parent_directory=None):
                 next_version.is_beta,
             )
         ):
-            next_version = VersionClass.parse("{}esr".format(bump_info["next_version"]))
+            next_version = VersionClass.parse("{}esr".format(next_version))
```

Comment:
I guess the intent is for this to saved_next_version? In any case, I would be inclined to spell this `next_version = attr.evolve(next_version, is_esr=True)` rather than parsing again.

Reply:
Either `next_version` should work at that stage. I didn't know about `attr.evolve`, it does seem more efficient, possibly less clear depending on familiar people are with it. Will have a think.